src/identity/tezos/TezosLedgerWallet.ts

```diff
@@ -59,6 +59,21 @@ export namespace TezosLedgerWallet {
         return signatureBytes;
     }
 
+    /**
+     * Signs arbitrary text using Ledger devices.
+     * 
+     * @param message UTF-8 test.
+     * @returns {Promise<string>} base58check-encoded signature prefixed with 'edsig'.
+     */
+    export async function signText(derivationPath: string, message: string): Promise<string> {
```

Comment:
Does this produce an identical signature to the one from the file wallet libSodium call?

Reply:
Yep